src/main/java/org/spongepowered/common/command/SpongeCommand.java

```diff
@@ -476,8 +479,16 @@ private static CommandSpec getPluginsCommand() {
 
                             appendPluginMeta(builder, "ID", container.getId());
                             appendPluginMeta(builder, "Description", container.getDescription());
-                            appendPluginMeta(builder, "URL", container.getUrl());
-
+                            appendPluginMeta(builder, "URL", container.getUrl().map(url -> {
+                                ClickAction.OpenUrl action = null;
+                                try {
+                                    // make the url clickable
+                                    action = TextActions.openUrl(new URL(url));
+                                } catch (MalformedURLException e) {
+                                    // or not
+                                }
+                                return Text.builder(url).onClick(action);
```

Comment:
:-1: for underlining, it makes it hard to read with a lot of resource packs, and partly with vanilla.